core/src/main/java/com/linecorp/armeria/server/file/DefaultMediaTypeResolver.java

```diff
@@ -72,6 +72,9 @@ enum DefaultMediaTypeResolver implements MediaTypeResolver {
         add(map, MediaType.create("application", "xml-dtd"), "dtd");
         add(map, MediaType.MANIFEST_JSON_UTF_8, "webmanifest");
 
+        // Compression formats
+        add(map, MediaType.create("application", "x-snappy-framed"), "sz");
```

Comment:
How about adding snappy to `MediaType`?

Reply:
Question: `br` and `gz` are handled somehow?

Reply:
I realized that we didn't need to add compressed formats here. The extensions are stripped and `MediaTypeResolver` tries to find the uncompressed format. Let me revert this change.